web_src/js/features/comp/EasyMDE.js

```diff
@@ -149,3 +152,19 @@ export function validateTextareaNonEmpty($textarea) {
   $mdeInputField.prop('required', false);
   return true;
 }
+
+/**
+ * @param {CodeMirror.EditorFromTextArea} codeMirror
+ */
+export function codeMirrorQuickSubmit(codeMirror) {
+  const textarea = codeMirror.getTextArea();
```

Comment:
What about doing the following:
```suggestion
 * @param {CodeMirror.EditorFromTextArea} codeMirror
 * @param {HTMLElement} callingTextArea non-CodeMirror text input to use for this function. Can be omitted.
 */
export function codeMirrorQuickSubmit(codeMirror, callingTextArea) {
  const textarea = callingTextArea ? callingTextArea : codeMirror.getTextArea();
```
If nothing is passed, the caller will be assumed to be codemirror, and otherwise, the given value will be used.
This has the benefit that the code duplication won't be needed anymore in the other method while not changing any behavior. Additionally reduces the number of bugs in the future as all text inputs then rely on the same implementation.